.github/workflows/jupyterlite.yml

```diff
@@ -0,0 +1,59 @@
+name: Build and Deploy Jupyterlite
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
+      - name: Install the dependencies
+        run: |
+          python -m pip install -r doc/jupyterlite/requirements.txt
+      - name: Build galgebra wheel
+        # NOTE: This is how we include the current github version of the package into the
+        # jupyterlite instance; if we used a different method, we would only get the version
+        # currently on PyPI or conda-forge.
+        run: |
+          python setup.py bdist_wheel
+          mkdir -p pypi  # Must be relative to cwd of the jupyterlite build step below
+          cp dist/*.whl pypi/
+      - name: Build the JupyterLite site
+        run: |
+          shopt -s globstar  # Allow ** below
+          mkdir -p doc/jupyterlite/content/examples
+          cp examples/**/*.ipynb doc/jupyterlite/content/examples
+          cp README.md doc/jupyterlite/content
+          jupyter lite build --contents doc/jupyterlite/content --output-dir jupyterlite_dist
+      - name: Upload artifact
+        uses: actions/upload-pages-artifact@v1
+        with:
+          path: ./jupyterlite_dist
```

Comment:
It's preferable to upload this to a url like https://pygae.github.io/galgebra/jupyterlite/ in case we need something on the root of https://pygae.github.io/galgebra/ .

I thought the doc was at this url, but actually it's on https://galgebra.readthedocs.io/ on Github Pages at the moment .

Reply:
Implemented and working nicely.